src/mavsdk/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -2367,6 +2367,22 @@ class Telemetry : public PluginBase {
      */
     std::pair<Result, Telemetry::GpsGlobalOrigin> get_gps_global_origin() const;
 
+    /**
+     * @brief Set rate to 'SYS_STATUS' updates.
+     * SYS_STATUS message allows to recieve vehicle HEALTH information.
+     * This function is non-blocking. See 'set_rate_sys_status' for the blocking counterpart.
+     */
+    void set_rate_sys_status_async(double rate_hz, const ResultCallback callback);
+
+    /**
+     * @brief Set rate to 'SYS_STATUS' updates.
+     * SYS_STATUS message allows to recieve vehicle HEALTH information.
+     * This function is non-blocking. See 'set_rate_sys_status_async' for the non-blocking counterpart.
+     * 
+     * @return Result of request.
+     */
+    Result set_rate_sys_status(double rate_hz) const;
+
```

Comment:
Note that this API is generated automatically from https://github.com/mavlink/MAVSDK-Proto/blob/main/protos/telemetry/telemetry.proto so the change needs to happen there first.

Sorry, I should have shared this earlier:
https://mavsdk.mavlink.io/main/en/cpp/contributing/plugins.html#add-api-to-proto